src/core/dev/cq_mgr_tx.cpp

```diff
@@ -151,6 +155,16 @@ bool cq_mgr_tx::request_notification()
     return true;
 }
 
+void cq_mgr_tx::reset_notification_armed()
```

Comment:
We are going to remove cq_mgr_tx eventually. You can move this code to the ring level.

Reply:
lets keep it and remove it in cleanup... i want to make one flow for dpcp and doca

Reply:
Its safe to do in ring_simple while keeping same flow

Reply:
Done.